github-cli.md

```diff
@@ -0,0 +1,110 @@
+**Quick Tip: Github and Command Line Interface (CLI)**
+======
+
+## **Overview**
+This is a quick guide to help you learn the basics about Github and CLI! If you have any more questions, or want other resources, [click here](https://www.atlassian.com/git/tutorials) for a good reference regarding Github and [ here](https://tutorial.djangogirls.org/en/intro_to_command_line/) for a good reference regarding CLI.
+
+## **Command Line Interface**
+<img src="https://memegenerator.net/img/instances/65841187/turn-off-the-gui-luke-use-the-command-line.jpg" width="300px" height="300px"/>
+
+### Introduction
+**What: ** CLI is a text-based interface which can be accessed through terminal (for OS X and Linux) or command prompt (for Windows). It allows a user to interact with the operating system seamlessly.
+
+**Why: ** While the Graphical User Interface (GUI) allows for editing and accessing files and folders, the CLI gives the user more power and precision.
+
+**NOTE: ** Some commands  are different for OS X and Linux vs Windows. Make sure you're using the right ones!
+
+### Commands
+
+#### Current Directory
+
+**OS X and Linux: ** `pwd` stands for "print working directory"
+
+**Windows: ** `cd` stands for "change directory" and can be used to switch to a new directory as well
+
+#### Learn More About a Command
+**OS X and Linux: ** `man <command>`
```

Comment:
- short for manual

Reply:
since this is the first time `<>` syntax has been used you may want to give an example as well, i.e. `man top`